Gems/Atom/Feature/Common/Code/Source/Shadows/ProjectedShadowFeatureProcessor.cpp

```diff
@@ -52,8 +59,15 @@ namespace AZ::Render
         desc.m_srgLayout = viewSrgLayout;
 
         m_filterParamBufferHandler = GpuBufferHandler(desc);
-        
-        CachePasses();
+
+        auto pipelines = GetParentScene()->GetRenderPipelines();
+        AZStd::for_each(pipelines.begin(), pipelines.end(),
+            [this](RPI::RenderPipelinePtr pipelinePtr)
+            {
+                CachePasses(pipelinePtr.get());
+            }
+        );
```

Comment:
nit, would something simpler be sufficient here?
```suggestion
        for(auto& renderPipeline: GetParentScene()->GetRenderPipelines())
        {
            CachePasses(renderPipeline->get());
        );
```

Reply:
yeah much better, I even wrote the loop that way in another place further down, not sure what I was thinking here.